hera_mc/cm_utils.py

```diff
@@ -194,13 +198,24 @@ def listify(X, None_as_list=False):
         else:
             return None
     if isinstance(X, str):
```

Comment:
`X` is not a good variable name in python. It should be multiple characters and not be capitalized per PEP8.